protostar/commands/test/cheatcodes/deploy_cheatcode.py

```diff
@@ -32,7 +33,15 @@ def name(self) -> str:
     def build(self) -> Callable[[Any], Any]:
         return self.deploy_prepared
 
-    def deploy_prepared(self, prepared: PreparedContract):
+    def deploy_prepared(
+        self,
+        prepared: PreparedContract,
+        *args,
+        # pylint: disable=unused-argument
+        config: Optional[Dict] = None,
+    ):
+        if len(args) > 0:
+            raise KeywordOnlyArgumentCheatcodeException(self.name, ["config"])
```

Comment:
Why did you modify `deploy_cheatcode`? This cheatcode doesn't have Migrator equivalent.